src/domains/movie.js

```diff
@@ -0,0 +1,63 @@
+const { PrismaClient } = require("@prisma/client");
+const prisma = new PrismaClient();
+
+class DomainMovie {
+  async fetchAllMovies() {
+    try {
+      return await prisma.movie.findMany({
+        include: {
+          screenings: true,
+        },
+      });
+    } catch (error) {
+      throw error;
+    }
```

Comment:
This `try/catch` is unnecessary because you're not doing anything with the error when you catch it, you could just run the prisma query and the same stuff would happen